website/draft_site/drafts/models.py

```diff
@@ -1,3 +1,38 @@
 from django.db import models
 
-# Create your models here.
+
+class Draft(models.Model):
+    num_drafters = models.IntegerField()
+    num_phases = models.IntegerField()
+    cards_per_pack = models.IntegerField()
+
+    def __str__(self):
+        return str(self.__dict__)
+
+
+class Drafter(models.Model):
+    draft = models.ForeignKey(Draft, on_delete=models.CASCADE)
+    seat = models.IntegerField()
+    # 0-indexed
+    current_phase = models.IntegerField(default=0)
+    # 0-indexed
+    current_pick = models.IntegerField(default=0)
+    bot = models.BooleanField()
+    # TODO: may need to store this outside DB eventually
+    bot_state = models.BinaryField(null=True, blank=True)
+
+    def __str__(self):
+        return 'ID: {}, Draft: {}, Seat: {}, Is Bot: {}, Phase: {}, Pick: {}'.format(
+            self.id, self.draft.id, self.seat, self.bot, self.current_phase, self.current_pick)
+
+
+class Card(models.Model):
+    draft = models.ForeignKey(Draft, on_delete=models.CASCADE)
+    name = models.CharField(max_length=50)
+    phase = models.IntegerField()
+    start_seat = models.IntegerField()
+    picked_by = models.ForeignKey(Drafter, on_delete=models.CASCADE, null=True, blank=True)
```

Comment:
Do you intend to have two different meanings for null and blank? Is there any reason to include both?

Reply:
See null and blank here: https://docs.djangoproject.com/en/2.2/ref/models/fields/#null